cli/tests/integrations/test_config.py

```diff
@@ -12,20 +12,24 @@
 
 @pytest.fixture
 def env():
-    return {
+    r = os.environ.copy()
```

Comment:
Yes, I assume that's the case for all the changes in this PR.  My question is *why* did it not work on Windows?